### DISCUSSION TITLE
Best type of stream to feed into frigate?

### DISCUSSION THREAD

**Discussion by boereworswizard** [mrlt8/docker-wyze-bridge#1395] [Q&A]
With Docker Wyze Bridge creating both WebRTC and RTSP streams, what's the best to feed into frigate?

**Comment by bpapa9013** [comment]
WebRTC is better, but trickier to setup in the frigate config file. Search the discussion and issues sections here and you will find several threads regarding some previous issues, and some setup examples. I think I found the best setup example in the frigate support discussions.

And to be 100% honest I'm not entirely sure if my frigate setup is actually using the go2rtc feeds built in to dwb or falling back to the standard RSTP feeds, I'm relatively new to docker, and very new to frigate and more advanced configuration of containers such as dwb/frigate/homeassistant. But I have been using dwb for a while just feeding the standard rtsp feeds into the nvr software Blue Iris.
